test/src/materials/ImplicitStateful.C

```diff
@@ -0,0 +1,53 @@
+/****************************************************************/
+/*               DO NOT MODIFY THIS HEADER                      */
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*           (c) 2010 Battelle Energy Alliance, LLC             */
+/*                   ALL RIGHTS RESERVED                        */
+/*                                                              */
+/*          Prepared by Battelle Energy Alliance, LLC           */
+/*            Under Contract No. DE-AC07-05ID14517              */
+/*            With the U. S. Department of Energy               */
+/*                                                              */
+/*            See COPYRIGHT for full restrictions               */
+/****************************************************************/
+#include "ImplicitStateful.h"
+
+template<>
+InputParameters validParams<ImplicitStateful>()
+{
+  InputParameters params = validParams<Material>();
+  params.addRequiredParam<std::string>("prop_name", "bla");
+  params.addRequiredParam<std::string>("coupled_prop_name", "bla");
+  params.addRequiredParam<bool>("add_time", "bla");
+  params.addRequiredParam<bool>("older", "bla");
+  return params;
+}
+
+ImplicitStateful::ImplicitStateful(const InputParameters & pars) :
```

Comment:
I thought we were favoring the newer get member function - especially since getParam doesn't necessarily return the same value.